src/collective/fingerpointing/testing.py

```diff
@@ -29,22 +32,37 @@ class Fixture(PloneSandboxLayer):
     defaultBases = (PLONE_FIXTURE,)
 
     # XXX: this can probably be done in a different way
-    def _configure_audit_log(self):
+    def _setup_audit_log(self):
         """Fake configuration for collective.fingerpointing as tests
         expect an audit.log file to be set.
         """
         from collective.fingerpointing.config import fingerpointing_config
-        fingerpointing_config['audit-log'] = '/tmp/audit.log'
+        self.temp_dir = tempfile.mkdtemp()
+        fingerpointing_config['audit-log'] = os.path.join(
+            self.temp_dir,
+            'audit.log',
+        )
+        from collective.fingerpointing.logger import log_info
+        log_info.configure(fingerpointing_config)
+
+    def _cleanup_audit_log(self):
+        from collective.fingerpointing.config import fingerpointing_config
```

Comment:
may I ask why you import this twice? just move it on top.

Reply:
In testing its always a good idea to import the modules as late and local as possible, but well, in the Fixture its not needed.

Reply:
But looking at the code (and not the diff) I had a reason, lets keep it this way. I think if there is a problem in the implementation it raises late